assets/quick-add.css

```diff
@@ -210,7 +213,13 @@ quick-add-modal .product-form__buttons {
   box-sizing: border-box;
 }
 
-quick-add-modal .product-media-container.constrain-height .media {
-  /* constrain behavior tbd, negate constrain effects on quick add modal for now */
-  padding-top: var(--ratio-percent);
+quick-add-modal .product-media-container.constrain-height {
+  --viewport-offset: calc(var(--modal-height-offset) + calc(var(--modal-padding) * 2) + calc(var(--popup-border-width) * 2));
```

Comment:
Shouldn't the `var(--modal-height-offset)` be `* 2` as well on mobile specifically ? Since we're applying that `--modal-heights-offset` value to the `margin-top` but also the `bottom` property 🤔